src/main/java/com/vinyl/controller/ProductController.java

```diff
@@ -3,20 +3,30 @@
 import com.vinyl.helper.AuthenticationHeaderHelper;
 import com.vinyl.modelDTO.AddProductToCartDTO;
 import com.vinyl.service.ProductService;
+import io.swagger.annotations.*;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+@Api(value="Product Management", description="Operations pertaining to products in Product Management System")
```

Comment:
please rename this to be more Vinyl store appropriate

Reply:
done